assets/css/base/_a.scss

```
@@ -1,16 +1,17 @@
 a {
-  border-bottom: $dark-grey solid 0.1em;
   color: $black;
   font-family: $ext-sans;
   margin: 1rem 0;
-  text-decoration: none;
+  text-decoration-color: $dark-grey;
+  text-decoration-line: underline;
+  text-decoration-thickness: 0.1em;
 
   &:hover,
   &:focus {
-    border-bottom: $primary-color solid 0.1rem;
+    text-decoration-color: $primary-color;
   }
   &:focus {
-    outline: pxToRem(3) solid $primary-color;
-    outline-offset: pxToRem(5);
+    background-color: pink;
+    color: $black;
   }
```

```suggestion
    outline: 0;
  }
```

@wuz I suggest we add in `outline:0` here. If we don't, the browser-default focus ring will still show, and we'll be right back to the Firefox issue that inspired this change. :)

---

Whoops! Great catch! Wonder why I didn't run into that when I was working on it. 🤷‍♂